APCAppCore/APCAppCore/UI/TabBarControllers/Activities/APCActivitiesViewSection.m

```diff
@@ -198,7 +198,7 @@ - (void) reduceToIncompleteTasksOnTheirLastLegalDay
                                                                                                  BOOL __unused *stopIterating) {
         BOOL keepThisOne = NO;
         
-        if (! taskGroup.isFullyCompleted && taskGroup.expiresToday)
+        if (! taskGroup.isFullyCompleted && [taskGroup expiresOnOrBeforeDate:self.today])
```

Comment:
should this still just check for expires today? will it end up showing incomplete tasks that expired on previous days? or will those somehow have already been filtered before it gets here